server/src/main/java/io/spine/server/aggregate/DefaultAggregatePartRepository.java

```diff
@@ -18,13 +18,35 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-package io.spine.system.server.given.command;
+package io.spine.server.aggregate;
 
-import io.spine.server.aggregate.AggregateRepository;
-import io.spine.system.server.CompanyId;
+import io.spine.server.aggregate.model.AggregatePartClass;
+
+import static io.spine.server.aggregate.model.AggregatePartClass.asAggregatePartClass;
 
 /**
- * A repository of {@link CompanyAggregate}.
+ * Default implementation of {@code AggregatePartRepository}.
```

Comment:
As discussed, we cannot follow this path, since `Repository` is not an interface. Let's make documentation backlinks to the `DefaultRepository` factory class. Together with the suggested `@Internal` annotations, it should explain the intent well.